src/providers/postgres/qgspostgresproviderconnection.cpp

```diff
@@ -354,6 +354,10 @@ QgsAbstractDatabaseProviderConnection::QueryResult QgsPostgresProviderConnection
           {
             vType = QVariant::DateTime;
           }
+          else if ( typName.startsWith( QLatin1String( "interval" ) ) )
+          {
+            vType = QVariant::fromValue( QgsInterval() ).type();
```

Comment:
This is always usertype?

Reply:
you are right. 
I've spotted another location where I made this one too.